org.kde.kstars.json

```diff
@@ -60,33 +70,106 @@
                     "x-checker-data": {
                         "type": "anitya",
                         "project-id": 270,
                         "stable-only": true,
-                        "url-template": "http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-$version.tar.gz"
+                        "url-template": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-$version.tar.gz"
                     }
                 }
             ]
         },
         {
-            "name": "libraw",
+            "name": "wcslib",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://src.rrz.uni-hamburg.de/files/src/wcslib/wcslib-7.10.tar.bz2",
+                    "sha256": "1796b0979df950ba7eae1010b986134067187846892b8e9b3c42d30361c9d929"
+                }
+            ]
+        },
+        {
+            "name": "stellarsolver",
+            "buildsystem": "cmake-ninja",
             "builddir": true,
-            "config-opts": [
-                "--disable-examples",
-                "--disable-static"
+            "config-opts": ["-DUSE_QT5=OFF"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rlancaste/stellarsolver/archive/refs/tags/2.7.tar.gz",
+                    "sha256": "36ff495f325920b9a23b7c3f7453a99c8d405e0984d4b27da7568625cb6c76c9",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 242699,
+                        "stable-only": true,
+                        "url-template": "https://github.com/rlancaste/stellarsolver/archive/refs/tags/$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "libev",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dist.schmorp.de/libev/Attic/libev-4.33.tar.gz",
+                    "sha256": "507eb7b8d1015fbec5b935f34ebed15bf346bed04a11ab82b8eee848c4205aea",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 1605,
+                        "stable-only": true,
+                        "url-template": "https://dist.schmorp.de/libev/Attic/libev-$version.tar.gz"
+                    }
+                }
             ],
+            "buildsystem": "autotools"
+        },
+        {
+            "name": "libnova",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://netix.dl.sourceforge.net/project/libnova/libnova/v%200.15.0/libnova-0.15.0.tar.gz?viasf=1",
+                    "sha256": "7c5aa33e45a3e7118d77df05af7341e61784284f1e8d0d965307f1663f415bb1"
+                }
+            ],
+            "buildsystem": "simple",
```

Comment:
this should be buildsystem autotools instead.

Reply:
can you paste the exact error when it fails? If it's because if missing autoreconf then you could use this little hack in sources, [example](https://github.com/flathub/org.gnome.Totem/blob/36c7d3224eb3f0b1e1870fb3d76c547938485689/codecs/libdvdcss.json#L11-L17)
```

        {
            "type":"script",
            "commands":[
                "autoreconf -fiv"
            ],
            "dest-filename":"autogen.sh"
        }
```

Reply:
this is not resolved. I don't see why it wouldn't work. But then you don't tell either.

Reply:
This flatpak itself does it:

https://github.com/flathub/org.kde.kstars/blob/master/org.kde.kstars.json#L102

No need to even look far.

(using a `shell` source is way simpler)